portal-api/src/v1/controllers/feedback.controller.js

```diff
@@ -60,7 +55,6 @@ exports.create = async (req, res) => {
     clearlyExplained,
     satisfied,
     howCanWeImprove,
-    submittedBy,
```

Comment:
I think we should be adding a users's username if it is available?

Reply:
since I changed to openRouter and removed validation, user does not show in the request anymore

Reply:
done

Reply:
Can't see the changes - also just thought, does the email get sent if there is no `submittedBy`? Might need to change the email template.

Reply:
forgot to push.  I have another ticket to fix the email template

Reply:
Does this mean the feedback email is broken until the other ticket is done?